### PR TITLE
Coach questions preview: Fix background and selection colors + resolve linter errors

### LINKED ISSUE #12425: EQM - Preview quiz - The selection effect is reversed
## Observed behavior

See the screenshot.

Screenshot:

selection

## Expected behavior

Only the currently selected question should be highlighted in light gray.

## Steps to reproduce the issue

1. Install the build from https://github.com/learningequality/kolibri/releases/tag/v0.17.0-beta0
2. Create a new quiz with sections and save it.
3. While at Coach > Quizzes click on the title of the quiz.

## Usage Details
Windows 11, Ubuntu 22  - Chrome

### PR DESCRIPTION
**Pull request by MisRob**
## Summary

- Fixes colors of list items in the question preview
- Remove hardcoded colors in favor of theme tokens
- Fixes linter errors (see commit messages)
- Removes unused component

**After (see the linked issue for 'before')**

after

## References

Closes https://github.com/learningequality/kolibri/issues/12425

## Reviewer guidance

Preview a quiz with several questions as a coach in Plan.

## Testing checklist

- [x] Contributor has fully tested the PR manually
- [x] If there are any front-end changes, before/after screenshots are included
- [ ] ~Critical user journeys are covered by Gherkin stories~
- [ ] ~Critical and brittle code paths are covered by unit tests~

## PR process

- [x] PR has the correct target branch and milestone
- [x] PR has 'needs review' or 'work-in-progress' label
- [x] If PR is ready for review, a reviewer has been added. (Don't use 'Assignees')
- [ ] ~If this is an important user-facing change, PR or related issue has a 'changelog' label~
- [ ] ~If this includes an internal dependency change, a link to the diff is provided~

## Reviewer checklist

- Automated test coverage is satisfactory
- PR is fully functional
- PR has been tested for [accessibility regressions](http://kolibri-dev.readthedocs.io/en/develop/manual_testing.html#accessibility-a11y-testing)
- External dependency files were updated if necessary (`yarn` and `pip`)
- Documentation is updated
- Contributor is in AUTHORS.md

### CODE REVIEW

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/QuestionListPreview.vue [hunk 1]
```diff
@@ -425,14 +425,6 @@
...
     height: 100%;
     padding: 0.5em;
-
-    &:hover {
-      background-color: white;
-    }
-
-    &.selected {
-      background-color: white;
-    }
```
Comment (nucleogenesis) [inline kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/QuestionListPreview.vue:435]: I did this in a bit of a rush 😅 - thanks for the reminder on $computedClass 😃

**Review by nucleogenesis — COMMENTED** [review]
Code changes LGTM - thanks Misha! I added @pcenov as a reviewer here to get his thoughts on the updates and if this sufficiently addresses the related issue

File: kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/QuestionListPreview.vue (hunk 1 above)
Comment (MisRob) [reply kolibri/plugins/coach/assets/src/views/plan/CreateExamPage/QuestionListPreview.vue:435]: No problem, can happen 🙂

**Review by pcenov — APPROVED** [review]
Thanks @MisRob - I confirm that https://github.com/learningequality/kolibri/issues/12425 is fixed.

**Comment by pcenov** [comment]
Hi @nucleogenesis I noticed that this fix is not in https://github.com/learningequality/kolibri/releases/tag/v0.17.0-beta2 - could you check why? Thanks!
cc @radinamatic

**Comment by radinamatic** [comment]
As far as I can see the fix is present in 0.17b3, so we should be good 🙂

**Comment by pcenov** [comment]
Hi @radinamatic and @nucleogenesis - the fix is still not present in [0.17.0-rc2](https://github.com/learningequality/kolibri/releases/tag/v0.17.0-rc2):

https://github.com/user-attachments/assets/d380940a-c989-45f0-9be0-0773681c1fb7

**Comment by pcenov** [comment]
@MisRob and @nucleogenesis - for some reason this is still not merged even in https://github.com/learningequality/kolibri/releases/tag/v0.18.0-alpha0 - could you investigate why is this the case?


https://github.com/user-attachments/assets/e5907630-606c-4e0d-aa3d-61e3ecb6f9c3

**Comment by MisRob** [comment]
@pcenov I wonder if it could be possibly caused a merge conflict, but I am not sure. I re-opened https://github.com/learningequality/kolibri/issues/12425.

**Pull request merged**